packages/app/src/debug/empty/DebugEmptyStates.cy.tsx

```diff
@@ -33,6 +33,10 @@ describe('Debug page empty states', () => {
       cy.findByRole('link', { name: 'Learn about project setup in Cypress' }).should('have.attr', 'href', 'https://on.cypress.io/adding-new-project?utm_source=Binary%3A+Launchpad&utm_medium=Debug+Tab&utm_campaign=Learn+More')
 
       cy.percySnapshot()
+
+      cy.viewport(600, 600)
+
+      cy.percySnapshot('responsive')
```

Comment:
I'm not seeing this snapshot in the [Percy build](https://percy.io/cypress-io/cypress/builds/24913506/) for some reason 🤔